sqlmesh/core/schema_loader.py

```
@@ -53,17 +53,26 @@ def create_schema_file(
         external_tables -= existing_models
 
     with ThreadPoolExecutor(max_workers=max_workers) as pool:
+
+        def _get_columns(table: str) -> t.Optional[t.Dict[str, t.Any]]:
+            try:
+                return adapter.columns(table, include_pseudo_columns=True)
+            except:
+                logger.warning(f"Unable to get schema for '{table}'")
```

you should log the exception and not just swallow it